mlir/test/CAPI/llvm.c

```diff
@@ -73,11 +79,47 @@ static void testTypeCreation(MlirContext ctx) {
           mlirTypeEqual(i32_i64_s, i32_i64_s_ref));
 }
 
+// CHECK-LABEL: testToLLVMIR()
+static void testToLLVMIR(MlirContext ctx) {
```

Comment:
This file is intended to test LLVM dialect APIs. Please create a new test file for the purposes of testing translation.